crates/wasi-nn/src/witx.rs

```diff
@@ -44,7 +44,10 @@ mod gen {
             match e {
                 WasiNnError::BackendError(_) => unimplemented!(),
                 WasiNnError::GuestError(_) => unimplemented!(),
-                WasiNnError::UsageError(_) => unimplemented!(),
+                WasiNnError::UsageError(e) => {
+                    println!("[OUCH!!!] {:?}", e);
```

Comment:
Let's just leave this unimplemented; we can follow up with better error handling in a separate PR.